.github/workflows/check-branch-commits.yaml

```diff
@@ -0,0 +1,34 @@
+# This workflow ensures that aptos-core branches do
+# not have inconsistent commits.
+
+name: "check-branch-commits"
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 9/12 * * *"
```

Comment:
please add a comment explaining what schedule this translates to in human language. Also note that the cron tab is interpreted in UTC so convert your human explanation to pacific time zone. See for example this one https://github.com/aptos-labs/aptos-core/blob/main/.github/workflows/find-packages-with-undeclared-feature-dependencies.yaml#L5

Reply:
Added a comment as taken directly from here: https://crontab.guru/#0_9/12_*_*_*. Note that we have a large number of jobs that also need to be updated in the repo.

Reply:
yeah you're right that technically the comment in the linked example should be 2am PST. I think when it was written we had daylight savings time so it actually translated to 3am (3am PDT technically).

While the daylight / standard switch in pacific time is kinda annoying (whereas UTC doesn't have that), I personally feel displaying this in pacific time (either PST or PDT) is still more readable, even if one may need to convert between PST/PDT in your mind.

Reply:
> While the daylight / standard switch in pacific time is kinda annoying (whereas UTC doesn't have that), I personally feel displaying this in pacific time (either PST or PDT) is still more readable, even if one may need to convert between PST/PDT in your mind.

Hmm, not sure I fully agree. If the time in the comment doesn't always directly relate to the reader, I'd rather just keep it simple, i.e., if I need to do a conversion from PDT to PST, I'd rather just convert once from UTC than have the PR author do UTC -> PDT, and then the reader do PDT -> PST. Plus, because crontab is already in UTC, it makes life easier. 😅  Either way, it's a small point, and if we want to standardize everything, there's a bunch of files that need to change.